CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+The Babelfish project is open source and we're happy to receive external
+contributions. But, as in any licensed project, there is some important legal
+information that you need to be aware of.
+
+First, most of the components of the Babelfish project are licensed under the GPU
+v3 or the Apache License 2.0. So, in order to contribute, you must accept to license
+your contribution using either one of those licenses, depending on the subproject
+(every project should have a LICENSE file with the specific license used).
+
+Also, we require that you read and accept our [Developer Certificate
+of Origin](https://developercertificate.org) which is exactly the same one that
+the Linux kernel uses. You can find a copy in this repository in the file `DCO` in
+the project root.  Please read it, but it basically says that you, for the best of
```

Comment:
It would be nice to have some source{d}-wide explanation (maybe here: https://github.com/src-d/guide/blob/master/engineering/licensing.md) but that goes beyond the scope of this PR.
If we provide legal interpretations, those should be provided in `src-d/guide` or any other place that is vetted by management.